srcpkgs/gcsf/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'gcsf'
+pkgname=gcsf
+version=0.1.17
+revision=1
+hostmakedepends="cargo rust pkg-config"
```

Comment:
`hostmakedepends="cargo pkg-config"`

Reply:
I guess cargo pulls the rust dependency. Will change.

Reply:
Yes, look at the template: `depends="rust"`
https://github.com/void-linux/void-packages/blob/master/srcpkgs/cargo/template#L7